global.json

```diff
@@ -1,5 +1,6 @@
 {
     "sdk": {
-      "version": "5.0.x"
+      "version": "5.0.100",
```

Comment:
This resolves version `5.0.302` on my machine (which is the latest version 5 installed on my machine).

Reply:
Ok